docs/getting-started/go.md

```diff
@@ -0,0 +1,239 @@
+# Getting Started with Go
+
+## Prerequisites
+
+- [Terraform](https://www.terraform.io/downloads.html) >= v0.12
+- [Node.js](https://nodejs.org) >= v12.16
+- [Go](https://golang.org/dl/) >= 1.11 (CDK for Terraform uses [go modules](https://golang.org/doc/go1.11#modules))
+
+### Install CDK for Terraform CLI
+
+Install with [Homebrew](https://brew.sh):
+
+```shell
+$ brew install cdktf
+```
+Or install with npm (requires [Node.js](https://nodejs.org)):
```

Comment:
Sounds like Nodejs would be optional. It's mandatory though

Reply:
Reworded for clarity